src/views/partials/breaches.js

```diff
@@ -118,5 +138,35 @@ export const breaches = data => `
     <span>${getMessage('column-detected')}</span>
   </header>
   ${createBreachRows(data.breachesData)}
+  <div class="no-unresolved-breaches-message">
+    <div class="no-breaches-message">
+      <img src='/images/breaches-none.svg' alt='' width="136" height="102" />
+      <h2>
+        ${getMessage('breaches-none-headline')}
+      </h2>
+      <p>
+        ${
+          data.breachesData.verifiedEmails.map(account => {
```

Comment:
Note that I'm just including the same string here multiple times - one for every of the user's email address. This is because I need to be able to toggle between the different strings on the client side, when the user selects a different email address, but of course becomes pretty cumbersome (or in some cases, even impossible) if we have to do this for every variable that changes on the client side. Do we have better solutions for this?

Reply:
Yeah, I see the difficulty.  I posted a follow up and tagged you for review: https://github.com/mozilla/blurts-server/pull/2828

I think using fetch for the dynamic content as @flozia suggested would have worked too, but in this case we were already tracking the selected email via module state, so it wasn't much more of a stretch to inject dynamically.

The client/partials/breaches.js module needed touch up anyway (and will need more after [MNTOR-1160/add-new-email](https://github.com/mozilla/blurts-server/tree/MNTOR-1160/add-new-email) lands).  Using `<template>` was not required, but I opted for it instead of dealing with toggling between div `hidden` states.